source/mainnet/net/guides/developer-page.rst

```diff
@@ -14,7 +14,7 @@ Official resources
 
 - See :ref:`SDKs and APIs<sdks-apis>` for links to SDKs and APIs for development
 
-- `Smart contracts <https://developer.concordium.software/en/mainnet/smart-contracts/index.html>`_
+- :ref:`Smart contracts <introduction>`
```

Comment:
It is weird that the smart contract introduction label is just introduction, which can be confusing if there other introductions.

Reply:
There are plenty of other introductions. Luckily the tools know exactly which one to insert.